tools/release/tests/test_release_remote.py

```diff
@@ -177,7 +177,7 @@ def test(remote_ip, version, token):
     ssh_execute(remote_ip, "sudo apt-get update", logger)
     ssh_execute(remote_ip, "sudo apt-get install -y gcc docker.io libyajl2 libseccomp2 libcap2", logger)
     ssh_execute(remote_ip, "sudo chmod 666 /dev/kvm", logger)
-    ssh_execute(remote_ip, f"sudo usermod -G docker {TESTING_VM_ADMIN}", logger)
+    ssh_execute(remote_ip, f"sudo usermod,kvm -G docker,kvm {TESTING_VM_ADMIN}", logger)
```

Comment:
Does "usermod,kvm" really work?  What does it do?  Am I mentally parsing it wrong?

Reply:
of course not. My stupidity ...